tests/dummy/app/components/date-time-test/template.hbs

```diff
@@ -1,6 +1,6 @@
 {{#nrg-form-container didValidate=didValidate as |form|}}
   {{#form.field label="Date" as |field|}}
-    {{field.datetime model=this valuePath="date" value=date}}
+    {{field.datetime model=this valuePath="date" value=value initializeDate=false}}
```

Comment:
`initializeDate` should probably default to true, but I wouldn't do it in the template. if you do it in the component, it would allow you to override it in the test when needed.